### ISSUE TITLE
Reporting complete log for training

### ISSUE THREAD

**Issue by tunai** [open-mmlab/mmdetection#486]
Greetings, 

I have been training a small dataset (60 samples w/ annotated BBs) using ResNet-50 as backbone and Cascade-RCNN as architecture. The training runs somewhat smoothly, but I can only see the validation results in the terminal, and when opening the log files, the same. 

I am starting the training wit the following: 
`./dist_train.sh /dir_for_my_config/cfg.py 1 --validate` (only 1 GPU)

After each epoch, I get validation results like: 
`+-------+-----+------+--------+-----------+-------+
| class | gts | dets | recall | precision | ap    |
+-------+-----+------+--------+-----------+-------+
| 1     | 8   | 0    | 0.000  | 0.000     | 0.000 |
+-------+-----+------+--------+-----------+-------+
| mAP   |     |      |        |           | 0.000 |
+-------+-----+------+--------+-----------+-------+
2019-04-11 11:12:22,729 - INFO - Epoch [6][29/29]       lr: 0.01410, mAP: 0.0000`

The log files look like this: 
`2019-04-09 11:40:54,937 - INFO - Start running, host: tunai@tunai-ubuntu2, work_dir: /dir
2019-04-09 11:40:54,937 - INFO - workflow: [('train', 1)], max: 12 epochs
2019-04-09 11:41:55,108 - INFO - Epoch [1][29/29]	lr: 0.00927, mAP: 0.0000
2019-04-09 11:42:57,244 - INFO - Epoch [2][29/29]	lr: 0.01023, mAP: 0.0000
2019-04-09 11:43:57,422 - INFO - Epoch [3][29/29]	lr: 0.01120, mAP: 0.0000
2019-04-09 11:44:56,876 - INFO - Epoch [4][29/29]	lr: 0.01217, mAP: 0.0000`

I am also printing region proposals, ground-truth BBs, etc. However, I am having a hard time calculating **the loss and accuracy** of the training set. And I have seen the logs of other projects in this forum, and these numbers are reported there. What am I missing? Are there any additional parameters in the dist_train.sh call? 

Thank you.

**Comment by hellock** [comment]
The log_interval is 50 by default, so the training log will be printed every 50 iterations. You need to change the interval.

**Comment by tunai** [comment]
Thank you hellock! That solved the problem.
